tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,33 @@
+const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+function golfScore(par, strokes) {
+  if (strokes == 1) {
+    return GOLF_SCORES.HOLE_IN_ONE;
+  } else {
+    let diff = strokes - par;
+    switch ( true ) {
+      case diff <= -2:
+        return GOLF_SCORES.EAGLE;
+      case diff == -1:
```

Comment:
use tripple equals check to avoid unexpected behaviour
You can add this rule to `.eslintrc.json` to avoid such mistake in future 
https://eslint.org/docs/rules/eqeqeq

Reply:
Thanks, I updated my ESLint file and replaced double equals with triple.